ui-tests/cypress/support/pageObjects/pages/templates/AddPage.js

```diff
@@ -55,36 +70,61 @@ export default class AddPage extends AppContent {
                .find(this.codeMirror);
   }
 
-  typeName(input) {
-    this.getNameInput().type(input);
+  getPreviewGrid() {
+    return this.getFormArea()
+               .find(this.previewGrid);
+  }
+
+  typeName(value) {
```

Comment:
Page object methods should return either an element of the page or the _currentPage_ itself; use the common method on the `Content` object to interact with the elements